.travis.yml

```diff
@@ -40,20 +34,20 @@ before_install:
   - pip install -U pip setuptools wheel
   - pip install --no-cache-dir -U -r requirements-test.txt
   - ./runcheckmigration
-  - flake8
-  - if [[ $TRAVIS_PYTHON_VERSION == 3* ]]; then ./runisort; fi
+  - if [[ $TRAVIS_PYTHON_VERSION != 3.8 ]]; then flake8; fi
+  - ./runisort
   - npm install -g jslint
 
 # command to install requirements
 install:
   - pip install $DJANGO
+  - pip install django-model-utils --upgrade
```

Comment:
don't do this, set the minimum version in the required dependencies if we need a specific minimum version.

Reply:
This is required else the build fails because one of the dependencies has django-model-utils in its dependencies but it installs a lower version of django-model-utils which further causes problem so this is required

Reply:
You have to put the following:

```
django-model-utils>=4.0
```

here: https://github.com/openwisp/django-x509/blob/master/requirements.txt#L3

So that version of django-model-utils should be automatically installed in the `install` step when openwisp-utils is installed.

Reply:
Its already there but in openwisp-utils `django-model-utils>=3.1.2,<3.3.0` is there so it is installing the previous version